core/src/com/mygdx/l5rdraft/packs/PackFactory.java

```diff
@@ -23,21 +23,7 @@ public static Pack createPack(List<Card> cards) {
             if (c.getRarity() == Card.RARITY.RARE) {
                 rares.add(c.copy());
             } else {
-                switch (c.getRarity()) {
-                    case SUPER_COMMON:
-                        count = 6;
-                        break;
-                    case COMMON:
-                        count = 3;
-                        break;
-                    case PROVINCE:
-                        count = 1;
-                        break;
-                    default:
-                        count = 0;
-                        break;
-                }
-                for (int i = 0; i < count; i++) {
+                for (int i = 0; i < c.getRarity().getCardCount(); i++) {
```

Comment:
adding the cardcount to the enum makes it so that you don't need this switch statement. Could be useful if more "rarities" (though unlikely) are added down the line. Looks a bit cleaner though.